applications/sckanner/frontend/src/components/connections/SummaryHeader.tsx

```diff
@@ -210,8 +210,8 @@ const SummaryHeader = ({
                 }}
               />
 
-              <Button variant="contained" onClick={generatePDF}>
-                Results Summary
+              <Button variant="outlined" onClick={generatePDF}>
```

Comment:
@Salam-Dalloul sorry I missed this, could you please fix this since the nomenclature has been changed recently to match what asked by the client? Open a follow up PR please. Thanks